### ISSUE TITLE
roc_curve crash

### ISSUE THREAD

**Issue by joos2010kj** [tianyu0207/RTFM#68]
https://github.com/tianyu0207/RTFM/blob/main/test_10crop.py#L26

pred = list(pred.cpu().detach().numpy())
pred = np.repeat(np.array(pred), 16)

 fpr, tpr, threshold = roc_curve(list(gt), pred)

Here, roc_curve crashes b/c gt's size and pred's size is inconsistent.

gt vs. pred size: [1114144, 1111824]

What would be the way to fix it?
